.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+	"root": true,
+	"extends": [
+		"wikimedia",
+		"wikimedia/node",
+		"@wmde/wikimedia-typescript"
+	],
+	"rules": {
+		"comma-dangle": [ "warn", {
+			"arrays": "always-multiline",
+			"objects": "always-multiline",
+			"imports": "always-multiline",
+			"exports": "always-multiline",
+			"functions": "always-multiline"
+		} ]
+	},
+	"overrides": [
+		{
+			"files": [ "**/*.ts" ],
+			"rules": {
+				"no-undef": "off"
```

Comment:
This is actually a valid question. I've added this so that eslint does not complain on things like `export default MwLog;` in `MwLog.ts` - Not a TS wizard, so not sure how commonly we might intend to use constructs like this. I've seen this a bunch of times in type definitions crafted by WMDE folks for Wikibase code, so assumed it probably is a common thing.
On the other hand, if this is only for this kind of uses, I figure using per-line ignores might be better. I'd be fine with going with what is simpler first (dunno if this one here, or explicit ignore line - thoughts?), and have a WMF+WMDE team decide on one.

Reply:
I guess this is a [known issue](https://github.com/eslint/typescript-eslint-parser/issues/416)!